modules/gdscript/gdscript_function.h

```diff
@@ -174,6 +174,20 @@ class GDScriptDataType {
 		return info;
 	}
 
+	Variant get_default_value() const {
+		Variant result;
+		if (has_type && kind == BUILTIN) {
+			Callable::CallError ce;
+			Variant::construct(builtin_type, result, nullptr, 0, ce);
+			ERR_FAIL_COND_V(ce.error != Callable::CallError::CALL_OK, Variant());
+			if (builtin_type == Variant::ARRAY && has_container_element_type()) {
+				Array array = result;
+				array.set_typed(container_element_type->builtin_type, container_element_type->native_type, container_element_type->script_type);
+			}
+		}
+		return result;
+	}
+
```

Comment:
Does a function like this not exist anywhere else? I thought I'd seen something like it. Perhaps it's worth making it a static `Variant` function?